maskrcnn_benchmark/modeling/roi_heads/box_head/roi_box_predictors.py

```diff
@@ -14,7 +14,8 @@ def __init__(self, config, pretrained=None):
         num_classes = config.MODEL.ROI_BOX_HEAD.NUM_CLASSES
         self.avgpool = nn.AvgPool2d(kernel_size=7, stride=7)
         self.cls_score = nn.Linear(num_inputs, num_classes)
-        self.bbox_pred = nn.Linear(num_inputs, num_classes * 4)
+        num_bbox_reg_classes = 2 if cfg.MODEL.CLS_AGNOSTIC_BBOX_REG else num_classes
```

Comment:
this is how Detectron implements it: by using 0 for background and 1 for the other classes.

But I'm starting to wonder if this is actually necessary here: given that we index out all negative examples via `sampled_pos_inds_subset`, we don't backprop through the background elements anyway.
Given that, I think we could have only `num_bbox_reg_classes = 1`, and the rest of the code should work as well.

@rbgirshick what do you think? Am I missing something here?